strawberryfields/gbs/data.py

```diff
@@ -103,7 +103,7 @@
 For example, we see that the ``data[3]`` sample has 11 clicks.
 
 Code details
-^^^^^^^^^^^^

```

Comment:
@josh146, what do you think of this change? Is there any problem to do this?

Reply:
Best to keep consistent; `===` for page titles, `---` for section headings, `^^^` for subheadings, `~~~`, etc.

Note that we normally have 'Code details' as a subheading, to avoid it showing in the sidebar.

Reply:
Thanks Josh! I actually don't mind "Code details" showing in the sidebar, and think it looks better in the rendered page to have it at the same depth as the other section headings.